hw/ip/usbdev/data/usbdev.hjson

```diff
@@ -22,6 +22,7 @@
     { name: "dn_pullup", desc: "USB D- pullup control" }
     { name: "tx_mode_se", desc: "USB single-ended transmit mode control" }
     { name: "suspend", desc: "USB link suspend state" }
+    { name: "d_input_enable", desc: "Enable signal for differential input buffer" }
```

Comment:
@vogelpi 
i think i would probably call this `rx_enable`, but not a big deal since this is bronze anyways :)

Reply:
another thing.. doesn't really matter for bronze, I don't think `d_input_enable` should be a pinmux signal since this one doesn't actually go to the pinmux.  It should probably be part of intersignal and directly fed out to the `ext` structure in top_earlgrey.hjson.  That way you don't need to add a dedicated IO (i think that's what's happening) for what is really an intermodule signal.

Reply:
yeah that's right, this should become an intermodule signal. I thought this does not matter much for Bronze, but let me amend that nevertheless.

Reply:
ok, made this an intermodule signal and renamed it to `usb_rx_enable`

Reply:
thanks @msfschaffner 
LGTM me otherwise.